conf/jitsi-videobridge.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=jitsi-videobridge
+After=network.target
+
+[Service]
+Type=simple
+User=__APP__
+Group=__APP__
+WorkingDirectory=__FINALPATH__/jitsi-videobridge/
+ExecStart=__FINALPATH__/jitsi-videobridge/jvb.sh --host=127.0.0.1 --domain=__DOMAIN__ --port=__PORT_COMPONENT__ --secret=__VIDEOBRIDGE_SECRET__ </dev/null >> /var/log/jvb.log 2>&1
```

Comment:
I never saw:

```
</dev/null >>
```

But, why not, do you know why adding this?

Reply:
Me neither, it's from the https://github.com/jitsi/jitsi-meet/blob/master/doc/manual-install.md#L178